bin/package/raspberry/files/default-myst-conf

```diff
@@ -0,0 +1,6 @@
+# Define additional args for `myst` service (see `myst --help` for full list)
```

Comment:
If we will delete `bin/package/raspberry/myststage`, GIT will track these files as renamed, but not new ones. 
We probably not need them anymore.

Reply:
Release build is not using gitlab yet, so I'm leaving it not to break compatibility.